app/services/search/postgres/podcast_episode.rb

```diff
@@ -0,0 +1,55 @@
+module Search
+  module Postgres
+    class PodcastEpisode
+      ATTRIBUTES = [
+        "podcast.id",
+        "podcast.image",
+        "podcast.published",
+        "podcast.slug",
+        "podcast_episodes.body",
+        "podcast_episodes.comments_count",
+        "podcast_episodes.id",
+        "podcast_episodes.podcast_id",
+        "podcast_episodes.processed_html",
+        "podcast_episodes.published_at",
+        "podcast_episodes.quote",
+        "podcast_episodes.reactions_count",
+        "podcast_episodes.subtitle",
+        "podcast_episodes.summary",
+        "podcast_episodes.title",
+        "podcast_episodes.website_url",
+      ].freeze
+      private_constant :ATTRIBUTES
+
+      DEFAULT_PER_PAGE = 60
+      private_constant :DEFAULT_PER_PAGE
+
+      MAX_PER_PAGE = 120 # to avoid querying too many items, we set a maximum amount for a page
+      private_constant :MAX_PER_PAGE
+
+      def self.search_documents(page: 0, per_page: DEFAULT_PER_PAGE, sort_by: nil, sort_direction: nil, term: nil)
+        # NOTE: [@rhymes/atsmith813] we should eventually update the frontend
+        # to start from page 1
+        page = page.to_i + 1
+        per_page = [(per_page || DEFAULT_PER_PAGE).to_i, MAX_PER_PAGE].min
+
+        relation = ::PodcastEpisode.includes(:podcast).where(podcast: { published: true })
```

Comment:
**blocking**: I think this is missing a condition, we display only `available` podcast episodes, which is a combination of `reachable: true` and the `podcast.published = TRUE` we have here. I think you can rewrite it as:

```ruby
relation = PodcastEpisode.available
```

without the rest

Reply:
@rhymes We don't use that filter in ES so I was going for 1:1. That said, good call, I think it makes sense to add it here. Updated the code, added a spec, and added an index for the heck of it!

Reply:
@atsmith813 oh! Didn't know! We use it for https://dev.to/pod - thanks for adding it. It'd be odd to display stuff there we don't display in the podcast page

Reply:
@rhymes Agreed! It almost seems like we meant to include/use `available` in ES...and maybe forgot(?). Either way, assuming we get the performance to a point we're happy with, this should be a solid search 💯